google_guest_agent/accounts_unix.go

```diff
@@ -12,6 +12,7 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
+//go:build !windows
```

Comment:
you can go ahead and remove these - we've fixed the presubmit to target our current golang version